coalib/settings/ConfigurationGathering.py

```diff
@@ -129,7 +129,8 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
 
     :param arg_list:    The list of command line arguments.
     :param log_printer: The LogPrinter object for logging.
-    :param arg_parser
+    :param arg_parser:  Instance of ArgParser that is used to
+                        parse none-setting arguments.
```

Comment:
Yes, I know. Sorry about that. Can you tell me which branch to delete and how to delete it ?